### ISSUE TITLE
automatic install using saas.py

### ISSUE THREAD

**Issue by BigWizzo** [it-projects-llc/odoo-saas-tools#653]
How do I use the saas.py for automatic install. I tried looking through the docs and I found nothing. please help.

**Comment by njeudy** [comment]
Hello,

what is the version you want to use ? 10.0 ? 11.0 ?

**Comment by BigWizzo** [comment]
Thanx for replying @njeudy I want to use version 11.0

**Comment by njeudy** [comment]
Ok I don't know if this script was migrated for now ( @yelizariev is it ? )
I did not use it personaly.

**Comment by BigWizzo** [comment]
Yes ( @yelizariev ). If you know how to use the version 10.0 please your help will be greatly appreciated @njeudy

**Comment by bitsnaps** [comment]
Does this script still work for v11?
